### PR TITLE
add kmala to the owners list

### PR DESCRIPTION
**Pull request by kmala**
**What this PR does / why we need it**:
Adding kmala to the owners list

contributions made: https://github.com/kubernetes-sigs/aws-iam-authenticator/pulls?q=is%3Apr+author%3Akmala

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -3,12 +3,14 @@ approvers:
 - wongma7
 - jyotimahapatra
 - nnmin-aws
```
Comment (bryantbiggs) [inline OWNERS:5]: can you remove Min's user name from the list here

Comment (kmala) [reply OWNERS]: done

**Pull request merged**